jdaviz/core/loaders/resolvers/resolver.py

```diff
@@ -279,8 +280,16 @@ def matches_target(importer):
 
     @observe('format_selected')
     def _on_format_selected_changed(self, change={}):
+        self.valid_import_formats = ''
```

Comment:
can this go in the else instead to avoid extra traitlet updates?  Or if its now static, can it be set in the init instead of in this observe?